package.json

```diff
@@ -16,7 +16,7 @@
   "main": "./lib/jake.js",
   "scripts": {
     "lint": "eslint --format codeframe \"lib/**/*.js\" \"test/**/*.js\"",
-    "prettier": "prettier --write \"{lib,test}/**/*.{js,ts}\"",
+    "lint:fix": "eslint --fix \"lib/**/*.js\" \"test/**/*.js\"",
```

Comment:
prettier was removed previously, so I replaced it with eslint-based solution along the way